lib/ansible/modules/infrastructure/stacki/stacki_host.py

```diff
@@ -0,0 +1,324 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2016, Hugh Ma <Hugh.Ma@flextronics.com>
+#
+# This module is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This software is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this software.  If not, see <http://www.gnu.org/licenses/>.
+
+DOCUMENTATION = '''
+---
+module: stacki_host
+short_description: Add or remove host to stacki front-end
+description:
+ - Use this module to add or remove hosts to a stacki front-end via API
+ - https://github.com/StackIQ/stacki
+version_added: "2.3"
+options:
+  chdir:
+    description:
+     - change working directory
+    required: false
+    default: null
+  cpu:
+    description:
+     - number of cpus to stress
+    required: true
+  delay:
+    description:
+     - piggybacks off the at module as a action plugin to run benchmark at a schedule time
+    required: false
+  executable:
+    description:
+     - path to stress executable if running from source
+    required: false
+  dest:
+    description:
+     - absolute path of file to write stdout to
+    required: false
+  timeout:
+    description:
+     - Total time to run stress for.
+    required: True
+requirements:
+ - requests
```

Comment:
`from ansible.module_utils.urls import (one of fetch_url, open_url)`

requests is a great library  ... that has not alway been great and mostly the installed versions are not great and many conflicts across versions why we tend to avoid it

Also, if anyone wants to use, they should really specify version
Developers often assume 'pip install requests' but most environments cannot do that